mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -596,23 +597,88 @@ impl TendermintCoin {
         sha256(&htlc_id).to_string().to_uppercase()
     }
 
+    pub(super) async fn seq_safe_send_raw_tx_bytes(
+        &self,
+        tx_payload: Any,
+        fee: Fee,
+        timeout_height: u64,
+        memo: String,
+    ) -> Result<(String, Raw), TransactionErr> {
+        let (tx_id, tx_raw) = loop {
+            let tx_raw = try_tx_s!(self.any_to_signed_raw_tx(
+                try_tx_s!(self.my_account_info().await),
+                tx_payload.clone(),
+                fee.clone(),
+                timeout_height,
+                memo.clone(),
+            ));
+
+            match self.send_raw_tx_bytes(&try_tx_s!(tx_raw.to_bytes())).compat().await {
+                Ok(tx_id) => break (tx_id, tx_raw),
+                Err(e) => {
+                    if e.contains("Wrong account sequence catched") {
```

Comment:
Can we make `Wrong account sequence catched` a const similar to `NO_TX_ERROR_CODE` https://github.com/KomodoPlatform/atomicDEX-API/blob/c97572a4e3908838e60a53ab8760ba59689679fc/mm2src/coins/utxo/rpc_clients.rs#L72